Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -295,12 +301,14 @@ void CGameListCtrl::Update()
                                         0);
     SetColumnWidth(COLUMN_BANNER,
                    SConfig::GetInstance().m_showBannerColumn ? 96 + platform_padding : 0);
-    SetColumnWidth(COLUMN_TITLE, 175 + platform_padding);
+    SetColumnWidth(COLUMN_TITLE, FromDIP(175) + platform_padding);
     SetColumnWidth(COLUMN_MAKER,
-                   SConfig::GetInstance().m_showMakerColumn ? 150 + platform_padding : 0);
-    SetColumnWidth(COLUMN_FILENAME,
-                   SConfig::GetInstance().m_showFileNameColumn ? 100 + platform_padding : 0);
-    SetColumnWidth(COLUMN_ID, SConfig::GetInstance().m_showIDColumn ? 75 + platform_padding : 0);
+                   SConfig::GetInstance().m_showMakerColumn ? FromDIP(150) + platform_padding : 0);
+    SetColumnWidth(COLUMN_FILENAME, SConfig::GetInstance().m_showFileNameColumn ?
+                                        FromDIP(100) + platform_padding :
+                                        0);
+    SetColumnWidth(COLUMN_ID,
+                   SConfig::GetInstance().m_showIDColumn ? FromDIP(75) + platform_padding : 0);
     SetColumnWidth(COLUMN_COUNTRY,
                    SConfig::GetInstance().m_showRegionColumn ? 32 + platform_padding : 0);
```

Comment:
Is there a particular reason why you `FromDIP` Title, Maker, FileName and ID, but not Platform, Banner, Country or EmulationState?